echo/index.js

```diff
@@ -181,13 +181,32 @@ function Client() {
     debug('request options', requestOptions);
 
     request.post(requestOptions, function onResp(err, response, body) {
-      var statusCode = response && response.statusCode
-        ? response.statusCode
-        : 0;
-
-      if (err || parseInt(statusCode / 100) !== 2) {
-        error('[echoClient] addEvents error', { err: err, body: body, statusCode: statusCode });
-        callback(err || 'error response status code: ' + statusCode);
+      if (err) {
+        var errorResponse = {
+          statusCode: 0,
+          message: err.message 
+        };
+        error(
+          '[echoClient] addEvents error', 
+          { 
+            error: errorResponse.message, statusCode: errorResponse.statusCode, body: body 
```

Comment:
It would be easier too read if each argument was on a separate line

Reply:
sure, will address